tests/testthat/test-utilities-figures.R

```diff
@@ -1,12 +1,51 @@
 ## context("esqLABS_colors")
+
+test_that("esqLABS_colors input validation works as expected", {
+  expect_error(esqLABS_colors(-1), "nrOfColors must be positive")
```

Comment:
As above - do not compare to hardcoded message.

Reply:
Done.